src/display/display-lazy-ads.ts

```diff
@@ -3,7 +3,7 @@ import { dfpEnv } from '../lib/dfp/dfp-env';
 import { enableLazyLoad } from './lazy-load';
 import { loadAdvert } from './load-advert';
 
-const instantLoadAdvertIds = ['dfp-ad--im'];
+const instantLoadAdvertIds: string[] = [];
```

Comment:
I can see this variable is being used on line 14 that is why I added an empty array but not 100% sure if we still need it or we can pass something else to partition as a second parameter?

Reply:
If there is no other slot using this I think we can remove this feature!

Reply:
Thank you Ravi for reviewing this and checking the code in our remove `im` catchup. This is now done 27a7758bffa8311c345c222a3702e50094033757